Cargo.toml

```diff
@@ -45,6 +45,7 @@ vello_shaders = { version = "0.3.0", path = "vello_shaders" }
 bytemuck = { version = "1.18.0", features = ["derive"] }
 skrifa = "0.22.3"
 peniko = "0.2.0"
+kurbo = "0.11.1"
```

Comment:
It required by the addtion of `kurbo = { workspace = true, optional = true }` in with_winit/Cargo.toml
without that it gets the following:

```
error: process didn't exit successfully: `~/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/bin/cargo metadata --format-version=1 --no-deps` (exit status: 101)
--- stderr
error: failed to load manifest for workspace member `~/git/linebender/vello/examples/with_winit`
referenced by workspace at `~/git/linebender/vello/Cargo.toml`

Caused by:
  failed to parse manifest at `~/git/linebender/vello/examples/with_winit/Cargo.toml`

Caused by:
  error inheriting `kurbo` from workspace root manifest's `workspace.dependencies.kurbo`
error: process didn't exit successfully: `~/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/bin/cargo metadata --format-version=1 --no-deps` (exit status: 101)
--- stderr
error: failed to load manifest for workspace member `~/git/linebender/vello/examples/with_winit`
referenced by workspace at `~/git/linebender/vello/Cargo.toml`

Caused by:
  failed to parse manifest at `~/git/linebender/vello/examples/with_winit/Cargo.toml`

Caused by:
  error inheriting `kurbo` from workspace root manifest's `workspace.dependencies.kurbo`

Caused by:
  `dependency.kurbo` was not found in `workspace.dependencies`
```

We could I suppose specify 0.11.1 in `with_winit` rather than picking it up from the workspace,
but the two versions (the one inherited through peniko, and the one specified explicitly) need to be kept in sync
to trigger the error (at least that is my belief, until a time where we have the feature supported in peniko itself).

Reply:
Yeah, this is just my mistake. I misunderstood where this was coming from.

Reply:
Perhaps at the very least we could add a comment specifying that the kurbo line can be removed once peniko supports the schemars feature and needs to be kept in sync with the version of kurbo used by peniko?

Reply:
It might also work to use a `"*"` version, but I don't have a good intuition about whether that will do what you want.

Reply:
Adding a `#[test]` definitely sounds like a good idea, will do.
I don't have a good intuition about whether a "*" version will do the right thing either though.

Reply:
I added that #[test] in ac97f1e, I don't think it will automagically run during ci.
but at the very least you can do so manually with `cargo test --features _ci_dep_features_to_test`
That said I did try to take care that the test doesn't actually require initializing gpu, etc.

Reply:
> Hmm would it not be automatically run in CI? It seems to be in your latest commit: https://github.com/linebender/vello/actions/runs/11730274415/job/32677779520#step:7:159

Well, I guess the following is enough for this test, since all we need to test is that it compiles.

`cargo clippy --tests --benches --examples --locked --no-default-features --features _ci_dep_features_to_test -- -D warnings`

But these appear to be running running clippy, not actually running any test target, that said I think it is sufficient?